client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -67,11 +67,10 @@ const IntroPricingBanner: FunctionComponent = () => {
 					/>
 					<span>
 						{ preventWidows(
-							translate( 'Get %(percent)d%% off your first year.*', {
+							translate( 'Get %(percent)d%% off your first year.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Get %(percent)d%% off your first year*' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/jetpack-pricing-asterisks)